### ISSUE TITLE
Remote SSH icon missing

### ISSUE THREAD

**Issue by wzm2256** [microsoft/vscode#109313]
- VSCode Version: 1.50.1
- OS Version: Windows 10
![image](https://user-images.githubusercontent.com/24223188/97098275-1b21f000-16b6-11eb-90a6-728e392a875c.png)

Steps to Reproduce:

1. Just install the Remote SSH extension. The remote SSH icon does not show up as expected.

Does this issue occur when all extensions are disabled?: Yes

I have disabled all other extensions as suggested, but the icon is still missing. This may be related to old bugs https://github.com/microsoft/vscode-remote-release/issues/687 and  https://github.com/microsoft/vscode-remote-release/issues/1407 which should have been solved already.

![image](https://user-images.githubusercontent.com/24223188/97098302-5ae8d780-16b6-11eb-9c91-3976cc0c3319.png)

The https://code.visualstudio.com/docs/remote/ssh-tutorial tutorial screen shot suggests the icon should show up like this:
![image](https://user-images.githubusercontent.com/24223188/97098456-27a74800-16b8-11eb-9105-5f1b0915eb36.png)

**Comment by wzm2256** [comment]
Finally, I solved it.
After reinstalling everything so many times.
For everyone who may have the same problem, I shared my solution here.

First of all uninstall VScode according to 
https://superuser.com/questions/1380208/how-to-completely-uninstall-visual-studio-code-from-windows-10

Then delete the following 3 folders if they exist (I'm not sure which one of them really causes this problem):

AppData/Local/Microsoft/vscode-cpptools
AppData/Local/TEMPT/vscode-typescript
AppData/Local/TEMPT/vscode-update-user-x64

Finally, install VSCode and everything is fine.
